# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/backdrop-filter-transformed-filter.yaml
# Commit: ea22ccce0c4d
# Full Hash: ea22ccce0c4dbe8a1c7ace807924275497f6c899
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-25 09:36:24
# Regressor Bug: 1769963
# File Overlap Count: 1
# Description:
#   Bug 1769963 - Fix backdrop-filter inside complex transform with CSS filter chain r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147124
# ==============================================================================

diff -r 60c64f026b6b -r ea22ccce0c4d gfx/wr/wrench/reftests/filters/backdrop-filter-transformed-filter.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/backdrop-filter-transformed-filter.yaml	Wed May 25 02:59:32 2022 +0000
@@ -0,0 +1,17 @@
+---
+root:
+  items:
+    - type: rect
+      color: red
+      bounds: 0 0 200 200
+    - image: checkerboard(2,14,14,14,14)
+      bounds: 0 0 200 200
+    - type: stacking-context
+      transform: rotate(45)
+      transform-origin: 100 100
+      filters: identity
+      wraps-backdrop-filter: true
+      items:
+        - type: backdrop-filter
+          bounds: 50 50 100 100
+          filters: invert(1)